.pre-commit-config.yaml

```diff
@@ -34,6 +34,12 @@ repos:
         language: system
         types:
           - python
+      - id: pydocstyle
+        name: pydocstyle
+        entry: pydocstyle
+        language: system
+        types:
+          - python
```

Comment:
Why we need this again? Because flake8 cant catch all errors?

Reply:
Initially, yes https://github.com/ComPWA/expertsystem/pull/28#pullrequestreview-411084795

But I found pydocstyle is also way better with Google (`napoleon`) conventions